### PR TITLE
Update Kusto version in insiders and main

### PR DESCRIPTION
**Pull request by gupta1**
This PR fixes #

### CODE REVIEW

**Review by joberume — APPROVED** [review]

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -3603,8 +3603,8 @@
 					},
 					"versions": [
 						{
-							"version": "0.1.0",
-							"lastUpdated": "9/1/2020",
+							"version": "0.1.1",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:3606]: You need to change the links below in these files too

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

**Comment by alanrenmsft** [comment]
also, have you already uploaded the new version of vsix to storage account?

File: extensionsGallery-insider.json (hunk 1 above)
Comment (gupta1) [reply extensionsGallery-insider.json:3606]: Made the url change as well.

**Comment by gupta1** [comment]
> also, have you already uploaded the new version of vsix to storage account?

Yes, its uploaded already. I will merge these changes only after few more verifications.

**Comment by alanrenmsft** [comment]
@gupta1 I am afraid you don't have permission to merge PR to release/extensions branch, just include me and Charles and we will merge it for you after approval.

**Review by alanrenmsft — APPROVED** [review]

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**